commands/endpoll.js

```diff
@@ -0,0 +1,61 @@
+const Discord = require('discord.js');
+
+const functions = require('../functions');
+
+module.exports = {
+	name: 'endpoll',
+	description: 'Close a poll using the poll channel and ID',
+	execute(client, message, args) {
+		if (args.length == 2) {
+			let guild = client.guilds.cache.get(client.CSIT_ID);
+			let channel = guild.channels.cache.find(ch => ch.name === args[0]);
+			if (channel != null) {
+				channel.messages.fetch(args[1])
+					.then(poll_msg => {
```

Comment:
You should definitely check out javascript's async/await. It really helps cleaning
up this callback hell you have here.

Reply:
Euch, promises are more reliable